content/reference-ledger-format.md

```diff
@@ -398,3 +398,72 @@ The `index` of a RippleState node is the SHA-512Half of the following values put
 * The AccountID of the low account
 * The AccountID of the high account
 * The 160-bit currency code of the trust line(s)
+
+
+## SignerList ##
+[[Source]<br>](https://github.com/ripple/rippled/blob/6d2e3da30696bd10e3bb11a5ff6d45d2c4dae90f/src/ripple/protocol/impl/LedgerFormats.cpp#L127 "Source")
+
+The `SignerList` node type represents a list of parties that, as a group, are authorized to sign a transaction in place of an individual account.
```

Comment:
I'm worried rephrasing it as you suggested makes "as a group" into a misplaced modifier which could seem to refer to the transaction, not the parties.

I'd prefer to leave this one as-is.